libs/tree-sitter-wing/bindings/rust/build.rs

```diff
@@ -7,12 +7,16 @@ fn main() {
 	generate_parser_in_directory(&PathBuf::from("."), None, tree_sitter::LANGUAGE_VERSION, false, None)
 		.expect("Generating parser");
 
+	// get WASI_SDK env
+	let wasi_sdk = std::env::var("WASI_SDK").expect("WASI_SDK env not set");
```

Comment:
Do we know this value at compile time? Maybe we can compile it inline https://doc.rust-lang.org/std/macro.env.html

Reply:
Nice, wasn't 100% sure it'd work due to this being in a build.rs but it looks like it's working